src/components/WithdrawlModal.tsx

```diff
@@ -1,77 +1,70 @@
-import React from 'react';
-import Button from '@material-ui/core/Button';
-import Input from '@material-ui/core/Input';
-import Dialog from '@material-ui/core/Dialog';
-import DialogActions from '@material-ui/core/DialogActions';
-import DialogContent from '@material-ui/core/DialogContent';
-import DialogContentText from '@material-ui/core/DialogContentText';
-import DialogTitle from '@material-ui/core/DialogTitle';
-import { Snackbar } from '@material-ui/core';
+import { ReactElement, useState } from 'react'
+import Button from '@material-ui/core/Button'
+import Input from '@material-ui/core/Input'
+import Dialog from '@material-ui/core/Dialog'
+import DialogActions from '@material-ui/core/DialogActions'
+import DialogContent from '@material-ui/core/DialogContent'
+import DialogContentText from '@material-ui/core/DialogContentText'
+import DialogTitle from '@material-ui/core/DialogTitle'
+import { Snackbar } from '@material-ui/core'
 
-import { beeDebugApi } from '../services/bee';
+import { beeDebugApi } from '../services/bee'
 
-export default function WithdrawlModal() {
-  const [open, setOpen] = React.useState(false);
-  const [amount, setAmount] = React.useState(BigInt(0));
-  const [showToast, setToastVisibility] = React.useState(false);
-  const [toastContent, setToastContent] = React.useState('');
+export default function WithdrawlModal(): ReactElement {
+  const [open, setOpen] = useState(false)
+  const [amount, setAmount] = useState(BigInt(0))
+  const [showToast, setToastVisibility] = useState(false)
+  const [toastContent, setToastContent] = useState('')
 
   const handleClickOpen = () => {
-    setOpen(true);
-  };
+    setOpen(true)
+  }
 
   const handleClose = () => {
-    setOpen(false);
-  };
+    setOpen(false)
+  }
 
   const handleWithdraw = () => {
     if (amount > 0) {
-        beeDebugApi.chequebook.withdraw(amount)
+      beeDebugApi.chequebook
+        .withdraw(amount)
         .then(res => {
-            setOpen(false);
-            handleToast(`Successful withdrawl. Transaction ${res.transactionHash}`)
+          setOpen(false)
+          handleToast(`Successful withdrawl. Transaction ${res.transactionHash}`)
```

Comment:
I am not a native english speaker and to me `withdrawl` seems a bit strange. I am familiar with `withdrawal` and it seems that is the more commonly used form.